x-pack/plugins/cases/public/components/app/types.ts

```diff
@@ -21,4 +21,5 @@ export interface CasesRoutesProps {
    */
   refreshRef?: MutableRefObject<CaseViewRefreshPropInterface>;
   timelineIntegration?: CasesTimelineIntegration;
+  onAlertsTableLoaded?: (eventIds: Array<Partial<{ _id: string }>>) => void;
```

Comment:
Could you explain why we are using `Partial<{ _id: string}>` here instead of `{ _id: string}`. Or is there a case where `_id` could not be defined?

Reply:
Being defensive is all I was thinking, as of now probably not, but maybe esql powered results come through some of these code paths soon, and didn’t want to break in that case.

Reply:
Gotcha! I think the use of `Partial` on an object type with just a single property threw me off.